text/0000-probability-grid-and-submap2d-restructuring-for-sdf.md

```diff
@@ -0,0 +1,36 @@
+# Generalize Probability Grid and Submap2D Structures for Signed Distance Field Approaches
+
+## Summary
+[summary]: #summary
+
+We propose to introduce a generalized structure of the `ProbabilityGrid` and `Submap2D` to allow the utilization of alternative grid data structures such as signed distance fields.
```

Comment:
SDF is one new option, a simple one would be to add intensities to the submaps and scan matching. You could mention this, also in the motivation.

Reply:
Please do. Also I would call it "alternative grid data representations".